pyproject.toml

```diff
@@ -59,6 +59,8 @@ dev = [
   "pytest",
   "tabulate",
   "mypy",
+  "python-semantic-release",
+  "pre-commit",
 ]
```

Comment:
since you have a `[dev]` and no `[ci]` option, you should replace `pip install .[ci]` by `pip install .[dev]`